### PR TITLE
Implement the Agent/FileName list handler, which will query information and upload to B2

### PR DESCRIPTION
**Pull request by askeing**
- [**IMPORTANT**] rename `ejenti.py` to `ejenti_cli.py`
  - due to `ejenti.py` is conflict with `ejenti` module path.
- add agents file name list handler for Trigger
- add parameter `--server-job-config` for `pulse_trigger`, which can load server jobs on Trigger machine
- support BlackBlaze API `b2_list_file_names` in `b2Util`
- add `server_job_config.json` template, and schema
- add unittest for `AgentsFileNameListHandler`

### CODE REVIEW

**Comment by askeing** [comment]
@ShakoHo @ypwalter r?

output example: http://jsoneditoronline.org/?id=61d4a2036c93790ba29951cf9b6ac23d

File: ejenti/pulse_trigger.py [hunk 1]
```diff
@@ -72,6 +74,13 @@ def main():
         logging.error('There is not command config. (Loaded from {})'.format(cmd_config_file))
         exit(1)
 
+    # loading server job config
+    server_job_config_arg = arguments['--server-job-config']
+    server_job_config_file = os.path.abspath(server_job_config_arg)
+    server_job_config = CommonUtil.load_json_file(server_job_config_file)
+    if not server_job_config:
+        logging.warn('There is not server job config, so, there is no server jobs. But trigger will still running. (Loaded from {})'.format(server_job_config_file))
```
Comment (ypwalter) [inline ejenti/pulse_trigger.py]: 'There is no server job config included. So, there will be no server job, but trigger will still go on.'

**Comment by ypwalter** [comment]
r+ after changing of warning message indicated in the file.

**Review by ypwalter — APPROVED** [review]

**Comment by askeing** [comment]
@ShakoHo @ypwalter @MikeLien @ycyen 

[**IMPORTANT**] 
This PR will rename `ejenti.py` to `ejenti_cli.py`.
All agents need to run `python setup.py install` before re-start `ejenti_cli`.

**Pull request merged**